spec/unit/views/components/table_for_spec.rb

```diff
@@ -301,6 +301,46 @@
       end
     end
 
+    context 'when i18n option is specified' do
+      before(:each) do
+        I18n.backend.store_translations(:en, {
+          activerecord: { attributes: { post: { title: "Name" } } }
+        })
```

Comment:
Indent the right brace the same as the first position after the preceding left parenthesis.